index.html

```diff
@@ -16,6 +16,76 @@
 </head>
 
 <body>
-<!-- Use your own code or past solution here! -->
+<div class='Main'>
+<div class='Main2'>
+      <header>
+            <nav>
+                  <a href="">Services</a>
+                  <a href="">Product</a>
+                  <a href="">Vision</a>
+                  <a href="">Features</a>
+                  <a href="">About</a>
+                  <a class='Connect' href="">Contact</a>
+            </nav>
+            <img src="img/logo.png" alt="logo">
+      </header>
+
+      <section class='Sec1'>
+            <div class='Sec1s1'>
```

Comment:
I know we haven't gone over BEM naming. But consider scanning the BEM page just to get a better understanding of how to name classes that are easy to decipher: http://getbem.com/naming/